thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/controller.js

```diff
@@ -320,9 +332,13 @@ export default Ember.Controller.extend({
   isEmailValid(emailArr) {
     const emailRegex = /^.{3,}\@linkedin.com$/;
     let isValid = true;
+
     for (var email of emailArr) {
```

Comment:
nit: this looks like a good candidate for a `reduce` (m to 1)

Reply:
actually I will use `emailArr.every(email => emailRegex.test(email))`